codegen_java/templates/libraries/okhttp-gson/build.sbt.mustache

```diff
@@ -39,7 +39,8 @@ lazy val root = (project in file(".")).
       "com.google.code.findbugs" % "jsr305" % "3.0.2" % "compile",
       "org.junit.jupiter" % "junit-jupiter-api" % "5.9.1" % "test",
       "com.novocode" % "junit-interface" % "0.10" % "test",
-      "org.mockito" % "mockito-core" % "3.12.4" % "test"
+      "org.mockito" % "mockito-core" % "3.12.4" % "test",
+      "org.json" % "json" % "20240205"
     ),
     dependencyOverrides += "org.json" % "json" % "20240205") ++ publishSettings:_*
```

Comment:
do you still need the dependencyOverrides?

Reply:
Lemme try without it.

Reply:
Looks like we don't. I'm wondering why `dependencyOverrides` didn't work, but I'm not so interested that I'm going to investigate it...

Reply:
it is probably an override for the local build but doesn't affect the pom, but I am just guessing